<launch>
  <arg name="world_frame"/>
  <arg name="hotend_group"/>
  <arg name="hotend_tcp"/>
  <arg name="hotend_base"/>
  <arg name="robot_model_plugin"/>
  <arg name="planner" default="ompl"/>

  <env name="ROSCONSOLE_CONFIG_FILE"
       value="$(find choreo_process_planning)/config/custom_rosconsole.conf"/>

  <node name="choreo_process_planning" pkg="choreo_process_planning" type="choreo_process_planning_node"
        output="screen">
    <param name="world_frame" value="$(arg world_frame)"/>
    <param name="hotend_group" value="$(arg hotend_group)"/>
    <param name="hotend_tcp" value="$(arg hotend_tcp)"/>
    <param name="hotend_base" value="$(arg hotend_base)"/>
    <param name="robot_model_plugin" value="$(arg robot_model_plugin)"/>
    <param name="planner" value="$(arg planner)" />
  </node>
</launch>
